### DISCUSSION TITLE
How does OXC compare to Bun?

### DISCUSSION THREAD

**Discussion by toughyear** [oxc-project/oxc#18] [Q&A] [answered]
OXC is interesting. Just wanted to know how this compares to other transpilers like "bun.sh". Is it a hobby project to learn Rust + compilation techniques?

**Comment by Boshen** [comment, marked as answer]
Thank you for asking.

From my understanding, Bun is a full runtime and also a company backed by VC.

This is a personal project in its infancy. What I want to achieve right now is to build the fastest JavaScript parser written in Rust. If this project gains enough attraction, I want to follow the path of the Rome Tools project and explore the possibility of building a linter and prettier.

(FYI the Rome Tools project is currently stagnant without any news.)

As you said it, I also want share my Rust knowledge and hopefully bring more JavaScript people over to Rust so everyone can learn from each other, so people can contribute to other RIIR (Rewrite it in Rust) projects.
